layers/debug_printf.cpp

```diff
@@ -242,11 +242,13 @@ std::string DebugPrintf::FindFormatString(std::vector<uint32_t> pgm, uint32_t st
     std::string format_string;
     SHADER_MODULE_STATE module_state(pgm);
     if (module_state.words_.size() > 0) {
```

Comment:
Is the size check needed? It seems strange to check the size before iterating through the loop, since normally the loop logic would handle that.

If it is needed return early.

```suggestion
    if (module_state.words_.empty()) { return {}; }
```